File: Łatwe/C++/Sumy wielokrotne/SumyWielokrotne.cpp
#include <iostream>

int main() {
    long long sumOfAllSums = 0LL;
    int number;
    while (std::cin >> number) {
        long long sum = 0LL;
        sum += number;
        while (number != 0 && std::cin >> number)
            sum += number;
        sumOfAllSums += sum;
        std::cout << sum << std::endl;
    }
    std::cout << sumOfAllSums;
}

File: Łatwe/Java/Sumy wielokrotne/SumyWielokrotne.java
import java.util.Scanner;

class SumyWielokrotne {
    public static void main(String[] args) {
        Scanner in = new Scanner(System.in);
        long sumOfAllSums = 0L;
        while (in.hasNextInt()) {
            long sum = 0L;
            int number = in.nextInt();
            sum += number;
            while (number != 0) {
                number = in.nextInt();
                sum += number;
            }
            sumOfAllSums += sum;
            System.out.println(sum);
        }
        System.out.print(sumOfAllSums);
    }
}
